system/Database/BaseBuilder.php

```diff
@@ -445,6 +445,22 @@ public function selectCount(string $select = '', string $alias = '')
         return $this->maxMinAvgSum($select, $alias, 'COUNT');
     }
 
+    /**
+     * Adds a subquery to the selection
+     */
+    public function selectSubquery(BaseBuilder $subquery, string $as): self
+    {
+        if (! $this->isSubquery($subquery)) {
```

Comment:
looking at mobile, it seems `$subquery` already type hinted, passed non `BaseBuilder` will already cause instance error so isSubquery() check no longer needed?

Reply:
You're right, this check was unnecessary